src/content/blog/2024-06-20-simple-open-payments-guide.md

```diff
@@ -0,0 +1,90 @@
+
+As an open standard, Open Payments is free for anyone to use, ensuring seamless communication between applications and their user’s account providers, regardless of which financial institutions are involved. This direct communication means you're not incurring additional fees or delays.
+
+The Open Payments protocol also aims to make transactions more transparent. Before committing to a payment, you understand how much the recipient is receiving and how much this payment costs you in transaction fees.
+
+The Open Payments standard addresses key questions such as:
+
+- Where should the money be sent, exactly? It provides a standardized way to identify and locate user accounts across different financial institutions.
+- Ensuring all parties agree on the final amount after any fees are added to a transaction.
+- Determining the best underlying method to use for the transaction.
+
+With the Open Payments standard, applications do not need to be registered financial service providers (FSPs) to facilitate transactions. Instead of moving or holding money, applications send payment obligations. This means they are not transferring money directly but are sending instructions to your financial institution to transfer the funds on your behalf. This setup eliminates the need for the application to be a financial service provider itself, even though it has direct communication access to your account. This is beneficial for applications because they don’t need to contend with the legal and compliance hurdles required to be a financial services provider. But it’s also advantageous for you because you don't need to share sensitive information with online applications. Applications storing information like credit card numbers and CVV codes always pose a risk of data leaks or hacking, even with the best intentions.
+
+You might think, this sounds convenient, but is it secure? I don’t want just anyone pulling money out of my account!
+
+## You Remain in Control
+
+With the Open Payments standard, you remain in full control of your financial transactions. When an application uses Open Payments, it securely shares key information about itself with the financial institutions it interacts with. This verification ensures that the account provider knows the application is legitimate when making a payment request on your behalf.
+
+Importantly, any withdrawal of money from your account requires your explicit consent. When you grant an application access to your account, you are not giving it unrestricted access. Instead, you specify the access rights: the exact amount, the time frame, and whether it can access your balance or move funds. Open Payments also supports recurring payments, allowing you to define how often, for how long, and up to what amount an application can access your account. This granular control ensures that you are always aware of and consenting to the transactions made from your account.
```

Comment:
> Instead, you specify the access rights: the exact amount, the time frame, and whether it can access your balance or move funds.

Maybe instead of you "specifying" the access rights, maybe using "control" is better, since the app/merchant would still be the one specifying amounts in the grant, for example. 

Also, instead of "access your balance", we could say access to your transaction history? Since balance isn't something we could know in Open Payments

Reply:
FWIW I like both of these suggestions